fix: the `raw` tokenizer no longer lowercases or truncates (#2445)
paradedb/paradedb: docs/documentation/indexing/tokenizers.mdx

);
```

### Keyword

> Treats the entire text as a single token, as-is. This tokenizer is required for `pg_search`'s query predicate pushdown

Reviewer: /
```suggestion
Treats the entire text as a single token, as-is. If a field is tokenized as `keyword`, using the `=` operator over that field can be pushed down to the index.
```

> support against fields that are `text` or `varchar`.

Reviewer: I think my suggestion messed up this sentence

> Treats the entire text as a single token, as-is. If a field is tokenized as `keyword`, using the `=` operator over that field can be pushed down to the index.
> support against fields that are `text` or `varchar`.

Author: ```suggestion
Treats the entire text as a single token, as-is. If a field is tokenized as `keyword`, using equality operators such as `=`, `<=`, `<`, `>, `>=`, `<>` with that field can be pushed down to the index.

The `keyword` tokenizer is supported for both `TEXT` and `VARCHAR` fields.
```
